fury/ui.py

```diff
@@ -3176,7 +3176,7 @@ def _get_actors(self):
         """ Get the actors composing this UI component.
         """
         actors = []
-        for option in self.options:
+        for option in self.options.values():
```

Comment:
You can't rely on the ordering of the elements in a Python dictionary. This iteration will depend on the PYTHONSEED and the machine. This means the actors will be added in arbitrary order.